apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/WireMockUtils.java

```diff
@@ -14,9 +14,18 @@ public record WireMockRuntime(WireMockRuntimeInfo runtimeInfo, WireMockServer se
     }
 
     public static WireMockRuntime startWireMock() {
-
         final WireMockServer wireMockServer = new WireMockServer(wireMockConfig().dynamicPort().dynamicHttpsPort());
 
+        return startWireMock(wireMockServer);
+    }
+
+    public static WireMockRuntime startWireMockHttpOnly() {
+        final WireMockServer wireMockServer = new WireMockServer(wireMockConfig().dynamicPort());
+
+        return startWireMock(wireMockServer);
```

Comment:
The previous server should give you both HTTP and https

Reply:
I'm receiving a certificate error for it. It works using http. If you have any idea how to make it work, I'm open for suggestions...

Reply:
Got it. You can use TestHttpClientUtils.client() The should solve your issue

Reply:
Done. Thanks!